### ISSUE TITLE
A question about your release of pretrained models

### ISSUE THREAD

**Issue by workingcoder** [lukemelas/EfficientNet-PyTorch#188]
I notice that you fixed the issue - '#177 Conv2dStaticSamePadding uses incorrect image size' (https://github.com/lukemelas/EfficientNet-PyTorch/issues/177) 24 days ago by modifying the code (https://github.com/lukemelas/EfficientNet-PyTorch/pull/179), which made your model have the correct size of feature maps.

But I think this modification will affect the head of the model when using global avg_pooling. And I notice that your latest pretrained model weights was released on 1 March before this modification (https://github.com/lukemelas/EfficientNet-PyTorch/releases). 

So I think you also need to fine tune the pretrained model weights to adapt to this modification to achieve the original classification accuracy level.

Thank you for your work and I will be looking forward to your new releases (^-^).
